### DISCUSSION TITLE
JSON error by using .CUE-sheets

### DISCUSSION THREAD

**Discussion by AndyFx12** [rern/rAudio#862] [Fixed Issues]
Adding a track from CUE-sheets (produced by ExactAudioCopy) throws 
`Error: JSON.parse: expected ',' or '}' after property value in object at line 47 column 21 of the JSON data{
  "player"       :▮ "mpd"
, "btclient"     : false
, "consume"      : false
, "control"      : "Digital"
, "counts"       : {
  "album": 836,
  "albumartist": 205,
  "artist": 3568,
  "composer": 110,
  "conductor": 4,
  "date": 99,
  "genre": 153,
  "playlists": 1,
  "latest": 1,
  "nas": 1,
  "sd": 0,
  "usb": 0,
  "song": 13455,
  "webradio": 84
}
, "icon"         : ""
, "librandom"    : false
, "relays"       : false
, "relayson"     : false
, "scrobble"     : false
, "stoptimer"    : false
, "stream"       : false
, "updateaddons" : false
, "updating_db"  : false
, "volume"       : 98
, "volumemute"   : 0
, "webradio"     : false
, "repeat" : false
, "random" : false
, "single" : false
, "file"           : "NAS/QNAP/Party/Apres_Ski_Megamix_2009-2CD-DE-2008-2Lions-Team/01-va_-_apres_ski_megamix_2009_cd1.cue/track0001"
, "playlistlength" : 6
, "song"           : 5
, "state"          : "play"
, "timestamp"      : 1648363427450
, "Album"  : "Apres Ski Megamix 2009"
, "Artist" : "Peter Wackel"
, "Time"   : 126
, "Title"  : "Ladioo (Arena-Hitmix)"
, "ext"      : "MP3""
, "coverart" : ""
, "icon"     : ""
, "sampling" : "6/6 • 24 bit 44.1 kHz 192 kbit/s • MP3""
, "elapsed"  : 110
, "coverart" : ""
}`
on playback tab.

status.sh | jq:
parse error: Invalid string: control characters from U+0000 through U+001F must be escaped at line 48, column 3

regards, Andy

**Comment by rern** [comment]
Please try the UPDATE branch.
By the way, post the `*.cue` file content to see where the extra `"`s after `"MP3""`came from.

**Comment by AndyFx12** [comment]
.cue-File:

PERFORMER "VA"
TITLE "Apres Ski Megamix 2009"
FILE "01-va_-_apres_ski_megamix_2009_cd1.mp3" MP3
  TRACK 01 AUDIO
    TITLE "Ladioo (Arena-Hitmix)"
    PERFORMER "Peter Wackel"
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "Michaela... Du Luder"
    PERFORMER "Willi Herren"
    INDEX 01 02:06:10
  TRACK 03 AUDIO
    TITLE "Resteficken"
    PERFORMER "Dj Abschleppdienst"
    INDEX 01 03:29:05
  TRACK 04 AUDIO
    TITLE "Get Ready to Bounce 2008"
    PERFORMER "Brooklyn Bounce"
    INDEX 01 05:44:21
  TRACK 05 AUDIO
    TITLE "Wir lieben es stürmisch (Heio Heio)"
    PERFORMER "Rühmanns (Sch)Erben"
    INDEX 01 07:45:70
  TRACK 06 AUDIO
    TITLE "Ra Ta Ta"
    PERFORMER "Mallorca Cowboys feat. Marco Mzee"
    INDEX 01 09:42:33
  TRACK 07 AUDIO
    TITLE "Ulli mit dem Pistenbully"
    PERFORMER "Strandjäger"
    INDEX 01 11:40:25
  TRACK 08 AUDIO
    TITLE "Ding Dong die Polizei"
    PERFORMER "Der Bademeister"
    INDEX 01 14:04:45
  TRACK 09 AUDIO
    TITLE "Ich will ein kind von dir"
    PERFORMER "Willi Girmes"
    INDEX 01 15:54:19
  TRACK 10 AUDIO
    TITLE "Yeke Yeke"
    PERFORMER "Picco"
    INDEX 01 17:57:49
  TRACK 11 AUDIO
    TITLE "Shut up And sleep with me"
    PERFORMER "Dj Ostkurve"
    INDEX 01 19:33:28
  TRACK 12 AUDIO
    TITLE "Fading like a Flower"
    PERFORMER "Dan Winter"
    INDEX 01 21:50:52
  TRACK 13 AUDIO
    TITLE "Everything is Changing"
    PERFORMER "Rikah"
    INDEX 01 23:48:70
  TRACK 14 AUDIO
    TITLE "Sex Toys"
    PERFORMER "Dj Lawless"
    INDEX 01 25:53:61
  TRACK 15 AUDIO
    TITLE "Neulich im Neandertal (Atte  katte nuva)"
    PERFORMER "Libero 5 feat. Dj Nüsse"
    INDEX 01 27:55:50
  TRACK 16 AUDIO
    TITLE "Das Kufsteiner Lied"
    PERFORMER "Fino"
    INDEX 01 29:33:49
  TRACK 17 AUDIO
    TITLE "Cotten Eye Joe"
    PERFORMER "Fun Tastic 3"
    INDEX 01 31:35:22
  TRACK 18 AUDIO
    TITLE "Reet Petite"
    PERFORMER "Jackie Wilson"
    INDEX 01 33:42:22
  TRACK 19 AUDIO
    TITLE "Wackel mit dem Arsch"
    PERFORMER "Chriss Tux und Dj Düse"
    INDEX 01 35:31:25
  TRACK 20 AUDIO
    TITLE "Wake up Little Susie"
    PERFORMER "THe Everly Brothers"
    INDEX 01 37:42:32
  TRACK 21 AUDIO
    TITLE "Westerland"
    PERFORMER "Tobee"
    INDEX 01 39:35:41
  TRACK 22 AUDIO
    TITLE "whole Lotta Shakin Goin On"
    PERFORMER "Jerry Lee Lewis"
    INDEX 01 42:24:23
  TRACK 23 AUDIO
    TITLE "Children of the Sun (R.I.O Radio mix)"
    PERFORMER "Yanou"
    INDEX 01 45:12:01
  TRACK 24 AUDIO
    TITLE "Davys On the Road again (Royal club radio mix)"
    PERFORMER "Royal Melody"
    INDEX 01 48:13:28
  TRACK 25 AUDIO
    TITLE "This life (Steve Mycs Radio edit)"
    PERFORMER "THe Vamprockerz"
    INDEX 01 50:28:23
  TRACK 26 AUDIO
    TITLE "Suavemente (M van Boyten & k la Cuard radio mix)"
    PERFORMER "Cuba Club"
    INDEX 01 52:17:02
  TRACK 27 AUDIO
    TITLE "Something (Phunk Foundation radio edit)"
    PERFORMER "Alex Megane"
    INDEX 01 53:46:74
  TRACK 28 AUDIO
    TITLE "A far Lamore Comincia Tu"
    PERFORMER "Bodybagers"
    INDEX 01 56:05:52
  TRACK 29 AUDIO
    TITLE "I Am Alive (Mondo Radio edit)"
    PERFORMER "Lazard"
    INDEX 01 57:50:49
  TRACK 30 AUDIO
    TITLE "Get this Party Started (dave ramone radio mix)"
    PERFORMER "Dan Winter"
    INDEX 01 59:48:57
  TRACK 31 AUDIO
    TITLE "The Concert Medley Seven Nation Army"
    PERFORMER "M@D"
    INDEX 01 61:54:19
  TRACK 32 AUDIO
    TITLE "So much love to give (spencer & hill radio mix)"
    PERFORMER "Dj Tom & Bump n Grind"
    INDEX 01 63:51:43
  TRACK 33 AUDIO
    TITLE "Rhythm of the night (ph electro radio edit)"
    PERFORMER "Verna"
    INDEX 01 65:52:45
  TRACK 34 AUDIO
    TITLE "Andy Borg (...Mir deine Lieder)"
    PERFORMER "Nickerbocker"
    INDEX 01 67:26:31
  TRACK 35 AUDIO
    TITLE "Verliebt, Verlor´n in Amarillo"
    PERFORMER "Mike Fender"
    INDEX 01 69:50:28
  TRACK 36 AUDIO
    TITLE "Dont bring me Down (lollies mix)"
    PERFORMER "Project N"
    INDEX 01 71:25:22
  TRACK 37 AUDIO
    TITLE "Ein bild von dir"
    PERFORMER "The Kingz"
    INDEX 01 73:17:73
  TRACK 38 AUDIO
    TITLE "Paloma Blanca (neuaufnahme)"
    PERFORMER "Nina & Mike"
    INDEX 01 74:37:70
  TRACK 39 AUDIO
    TITLE "Michaela (neuaufnahme)"
    PERFORMER "Bata Illic"
    INDEX 01 76:49:74

Fix is working fine! Thanks!
